app/layout.tsx

```diff
@@ -1,11 +1,11 @@
-import "./globals.css";
-import { Inter } from "next/font/google";
+import './globals.css';
+import {Inter} from 'next/font/google';
 
 const inter = Inter({ subsets: ["latin"] });
 
 export const metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+  title: "BAGGER",
+  description: "De website van Studenten Dispuut Voor Alle Studies Maar Vooral Voor Informatie en Communicatie Technologische Soorten Studies BAGGER",
```

Comment:
"Website der Studenten Dispuut Voor Alle Studies Maar Vooral Voor Informatie- en Communicatie Technologische Soorten Studies BAGGER" is misschien ook wel lekker fake-prominent